### PR TITLE
Upgrade to Go 1.19

### PR DESCRIPTION
**Pull request by johanbrandhorst**
Go 1.19 includes new behavior for `go fmt` and other tools that depend on its behavior (like `goimports` and `gofumpt`).

### CODE REVIEW

File: internal/auth/oidc/repository_auth_method_read.go [hunk 1]
```diff
@@ -28,7 +28,8 @@ func (r *Repository) LookupAuthMethod(ctx context.Context, publicId string, opt
 
 // ListAuthMethods returns a slice of AuthMethods for the scopeId. The
 // WithUnauthenticatedUser, WithLimit and WithOrder options are supported and
-//  all other options are ignored.
```
Comment (louisruch) [inline internal/auth/oidc/repository_auth_method_read.go:31]: The extra space here was an error

File: internal/daemon/controller/handlers/filtering.go [hunk 2]
```diff
@@ -33,7 +33,8 @@ func NewFilter(f string) (*Filter, error) {
 // Match returns if the provided interface matches the filter.
 // If the filter does not match the structure of the object being Matched, false is returned.
 // TODO: Support more than just matching against the item being filtered.  Also allow matching against
-//   values in the request or the request context itself.
```
Comment (louisruch) [inline internal/daemon/controller/handlers/filtering.go:36]: same re extra space triggering a list

File: internal/db/schema/internal/postgres/postgres.go [hunk 3]
```diff
@@ -422,7 +422,9 @@ func (p *Postgres) EnsureMigrationLogTable(ctx context.Context) error {
 
 // GetMigrationLog will retrieve the migration logs from the db for the last
 // migration.
-//  The WithDeleteLog option is supported and will remove all log entries,
```
Comment (louisruch) [inline internal/db/schema/internal/postgres/postgres.go:425]: space

File: internal/db/schema/migration_log.go [hunk 4]
```diff
@@ -19,7 +19,8 @@ type LogEntry struct {
 
 // GetMigrationLog will retrieve the migration logs from the db for the last
 // migration. Once it's read the entries, it will delete them from the database.
-//  The WithDeleteLog option is supported and will remove all log entries when provided.
```
Comment (louisruch) [inline internal/db/schema/migration_log.go:22]: space

**Review by louisruch — CHANGES_REQUESTED** [review]
Found a couple places where it seems someone was putting comments on a newline and included a leading space. Other than that is this great thanks!

**Comment by jefferai** [comment]
I suspect the author of some comment wrapping plugin is going to get a strongly worded tweet :-)

**Comment by johanbrandhorst** [comment]
I've fixed the whitespace errors (and a few new ones), please take another look!

**Review by louisruch — APPROVED** [review]
LGTM

**Pull request merged**